v2/esp-entity-table.ts

```diff
@@ -24,6 +24,13 @@ interface entityConfig {
   speed: string;
 }
 
+function getBasePath() {
+  let str = window.location.pathname;
+  return str.endsWith("/") ? str.slice(0, -1) : str;
+}
```

Comment:
Why duplicate this and not just put it somewhere common?

Reply:
Fair point. 
Cleaned it up using an exported function.